support/templates/partials/treatment.hbs

```diff
@@ -56,26 +56,37 @@
         <p lang="nl-BE" property="besluit:onderwerp">{{this.subject}}</p>
         {{> mandateeList mandatees=this.attendees property="besluit:heeftAanwezige" title="Aanwezigen tijdens de stemming" includeRole=false}}
         {{> mandateeList mandatees=this.voters property="besluit:heeftStemmer" title="Effectieve stemmers" includeRole=false}}
-        <p>
-          Totaal aantal voorstanders:
-          <span datatype="xsd:integer" property="besluit:aantalVoorstanders" content="{{this.positiveVotes}}">{{this.positiveVotes}}</span>
-        </p>
-        <p>
-          Totaal aantal tegenstanders:
-          <span datatype="xsd:integer" property="besluit:aantalTegenstanders" content="{{this.negativeVotes}}">{{this.negativeVotes}}</span>
-        </p>
-        <p>
-          Totaal aantal onthoudingen, blanco of ongeldig:
-          <span datatype="xsd:integer" property="besluit:aantalOnthouders" content="{{this.abstentionVotes}}">{{this.abstentionVotes}}</span>
-        </p>
-        {{#unless this.geheim}}
-          {{> mandateeList mandatees=this.positiveVoters property="besluit:heeftVoorstander" title="Voorstanders" includeRole=false}}
-          {{> mandateeList mandatees=this.negativeVoters property="besluit:heeftTegenstander" title="Tegenstanders" includeRole=false}}
-          {{> mandateeList mandatees=this.abstentionVoters property="besluit:heeftOnthouders" title="Onthoudingen, blanco of ongeldig" includeRole=false}}
-        {{/unless}}
-        <p>
-          Met als gevolg,
-        </p>
+        {{#if this.isUnanimous}}
+          <span datatype="xsd:integer" property="besluit:aantalVoorstanders" content="{{this.positiveVotes}}"></span>
+          <span datatype="xsd:integer" property="besluit:aantalTegenstanders" content="{{this.negativeVotes}}"></span>
+          <span datatype="xsd:integer" property="besluit:aantalOnthouders" content="{{this.abstentionVotes}}"></span>
+          {{#unless this.geheim}}
+            {{> mandateeList mandatees=this.positiveVoters property="besluit:heeftVoorstander" title="Voorstanders" includeRole=false hideText=true}}
+            {{> mandateeList mandatees=this.negativeVoters property="besluit:heeftTegenstander" title="Tegenstanders" includeRole=false hideText=true}}
+            {{> mandateeList mandatees=this.abstentionVoters property="besluit:heeftOnthouders" title="Onthoudingen, blanco of ongeldig" includeRole=false hideText=true}}
+          {{/unless}}
+          <p>Met eenparigheid van stemmen,</p>
```

Comment:
This also needs to specifiy if it was a positive unanimous vote or a negative one. so:
"De raad stemt voor, met eenparigheid van stemmen" (positive)
or
"De raad stemt tegen, met eenparigheid van stemmen" (negative)

Reply:
I don't really agree, this would be clear from the result ("gevolg") I think.